entries/214_show_notes.bacf

```diff
@@ -0,0 +1,92 @@
+- nr: 214
+- title: Donny Wals’ Advent of Swift, Disney+ and other things - A special Christmas Episode
+- date: 17.10.19, 20:00
+- file: 214_advent_of_swift.mp3
+- author: Bas Broek / Benedikt Terhechte / Donny Walls
+- description: TBD
+---
+<div>
+<p>
+  Please let us know your take-away of this episode by sharing it with us <a href="https://twitter.com/_contravariance">@_contravariance</a>.
+</p>
+<p>
+  An <a href="https://itunes.apple.com/us/podcast/contravariance-a-swift-podcast/id1423771323">iTunes review</a> is also greatly appreciated!
+</p>
+<p>
+  <h3>Contact information</h3>
+
+  <ul>
+    <li><a href="https://contravariance.rocks">Contravariance's website</a></li>
+    <li><a href="https://github.com/Contravariance/website">GitHub repository</a></li>
+    <li><a href="https://twitter.com/_contravariance">Contravariance's Twitter</a></li>
+    <li><a href="https://twitter.com/terhechte">Benedikt's Twitter</a></li>
+    <li><a href="https://twitter.com/basthomas">Bas' Twitter</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="https://appventure.me">Benedikt's blog</a></li>
+    <li><a href="https://basthomas.github.io">Bas' blog</a></li>
+  </ul>
+</p>
+<p>
+  <h3>Introduction</h3>
+
+  <ul>
+  	<li><a href="https://twitter.com/twannl">Antoine van der Lee</a></li>
```

Comment:
Looks like the notes are from the last show :)

Reply:
Yep! The process is I edit the audio, and Bas edits the notes